### ISSUE TITLE
How to run image_classifier_demo?

### ISSUE THREAD

**Issue by anshumang** [milakov/nnForge#19]
I have downloaded the ImageNet dataset from the Googe Drive location mentioned in the readme. 
The default action of 'demo' doesn't work because I am running on a server.
I tried 'train' but that gives the following error-
Exception caught: basic_ios::clear

Full log -
Forking output log to /opt/share/users/anshuman/nnForge/working_data/image_classifier_demo/log.txt...
########################################
2015-02-11 12:06:39
action=train
input_data_folder="/opt/share/users/anshuman/nnForge/input_data/image_classifier_demo"
working_data_folder="/opt/share/users/anshuman/nnForge/working_data/image_classifier_demo"
ann_count=1
training_epoch_count=50
snapshot_count=100
snapshot_extension=jpg
snapshot_extension_video=avi
snapshot_scale=1
snapshot_video_fps=5
snapshot_ann_index=0
snapshot_ann_type=image
learning_rate=0.02
learning_rate_decay_tail=0
learning_rate_decay_rate=0.5
learning_rate_rise_head=0
learning_rate_rise_rate=0.1
batch_offset=0
test_validate_ann_index=-1
snapshot_data_set=training
profile_updater_entry_count=1
check_gradient_weights=::
check_gradient_threshold=1.05
check_gradient_base_step=0.001
training_algo=sgd
dump_resume=1
load_resume=0
epoch_count_in_training_set=1
weight_decay=0
batch_size=1
momentum=0
shuffle_block_size=-1
report_stats=0
cuda_max_global_memory_usage_ratio,G=0.9
cuda_device_id,D=0
----------------------------------------
Exception caught: basic_ios::clear
########################################

**Comment by milakov** [comment]
Hi, sorry to say, this partcular sample is demo for inference only; it has no training implemented. You downloaded trained model.
